components/user-store/org.wso2.carbon.identity.user.store.configuration/src/main/java/org/wso2/carbon/identity/user/store/configuration/UserStoreConfigService.java

```diff
@@ -118,4 +120,16 @@ boolean testRDBMSConnection(String domainName, String driverName, String connect
      */
     void modifyUserStoreState(String domain, Boolean isDisable, String repositoryClass)
             throws IdentityUserStoreMgtException;
+
+    /**
+     * Get all userstores of the given repository.
+     *
+     * @return An array of {@link UserStoreDTO}
+     * @throws IdentityUserStoreServerException Throws an error when getting user store attribute mappings
```

Comment:
missing full stops at the end

Reply:
fixed in https://github.com/wso2/carbon-identity-framework/pull/3715